syft/pkg/cataloger/cpe.go

```diff
@@ -359,7 +322,12 @@ func vendorsFromGroupIDs(groupIDs []string) *strset.Set {
 			}
 
 			// e.g. jenkins-ci -> [jenkins-ci, jenkins]
-			vendors.Add(generateSubSelections(field)...)
+			for _, value := range generateSubSelections(field) {
```

Comment:
this should be commented

Reply:
this has been removed here https://github.com/anchore/syft/pull/484#discussion_r702009614